yt_dlp/dependencies/__init__.py

```diff
@@ -43,6 +43,8 @@
 
 try:
     import sqlite3
+    # `sqlite3.version` is deprecated in py3.12, see https://github.com/yt-dlp/yt-dlp/issues/8152
```

Comment:
I would maybe explain why we are using `sqlite_version` instead. Currently this doesn't help understand that `sqlite3.version` is useless. 
```suggestion
    # We want to get the underlying `sqlite` version, see https://github.com/yt-dlp/yt-dlp/issues/8152
```